packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.sqs.ts

```diff
@@ -0,0 +1,73 @@
+import { HttpMethod, WebSocketApi, WebSocketStage } from 'aws-cdk-lib/aws-apigatewayv2';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { App, Stack, Aws } from 'aws-cdk-lib';
+import { WebSocketAwsIntegration } from 'aws-cdk-lib/aws-apigatewayv2-integrations';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+/*
+ * Stack verification steps:
+ * 1. Verify manually that the integration has type "MOCK"
+ */
+
+const app = new App();
+const stack = new Stack(app, 'integ-aws-websocket-sqs-integration');
+
+const sqsMessageQueue = new sqs.Queue(stack, 'MessageSQSQueue', {
+  fifo: true,
+  queueName: 'MessageSQSQueue.fifo',
+});
+
+// API Gateway WebSocket API
+const webSocketApi = new WebSocketApi(stack, 'webSocketApi', {
+  description: 'Send websocket data to SQS which is then processed by a Lambda 2',
+  routeSelectionExpression: '$request.body.action',
+});
+
+// Optionally, create a WebSocket stage
+new WebSocketStage(stack, 'DevStage', {
+  webSocketApi: webSocketApi,
+  stageName: 'dev',
+  autoDeploy: true,
+});
+
+// IAM Role for API Gateway
+const webSocketApiRole = new iam.Role(stack, 'webSocketApiRole', {
+  assumedBy: new iam.ServicePrincipal('apigateway.amazonaws.com'),
+});
+
+webSocketApiRole.addToPolicy(
+  new iam.PolicyStatement({
+    actions: ['sqs:SendMessage'],
+    effect: iam.Effect.ALLOW,
+    resources: [sqsMessageQueue.queueArn],
+  }),
+);
+
+webSocketApi.addRoute('$default', {
+  integration: new WebSocketAwsIntegration('SQSSendMessage', {
+    integrationUri: `arn:aws:apigateway:${Aws.REGION}:sqs:path/${Aws.ACCOUNT_ID}/${sqsMessageQueue.queueName}`,
+    integrationMethod: HttpMethod.POST,
+    credentialsRole: webSocketApiRole,
+    templateSelectionExpression: '\\$default',
+    requestTemplates: {
+      /*eslint quotes: ["error", "single", { "avoidEscape": true }]*/
```

Comment:
For my understanding, what is this comment for?

Reply:
Hi Gavin, it just allows you to avoid escapes when using single and double quotes for literals. You can certainly escape the single quotes if desired and remove the comments. https://eslint.org/docs/latest/rules/quotes